[arg] [0]=/HOME/scz1075/.conda/envs/kungfu/bin/python
[arg] [1]=/HOME/scz1075/.local/bin/kungfu-run
[arg] [2]=-np
[arg] [3]=2
[arg] [4]=/HOME/scz1075/.conda/envs/kungfu/bin/python3
[arg] [5]=tf_cnn_benchmarks.py
[arg] [6]=--batch_size=64
[arg] [7]=--data_format=NCHW
[arg] [8]=--model=resnet50
[arg] [9]=--optimizer=momentum
[arg] [10]=--variable_update=kungfu
[arg] [11]=--kungfu_option=sync_sgd
[arg] [12]=--nodistortions
[arg] [13]=--num_gpus=1
[arg] [14]=--num_epochs=90
[arg] [15]=--weight_decay=1e-4
[arg] [16]=--data_dir=/HOME/scz1075/run/data/cifar-10-batches-py
[nic] [0] lo :: 127.0.0.1/8, ::1/128
[nic] [1] enp39s0f3u1u2c2 :: 
[nic] [2] eno1 :: 10.254.24.56/22, fe80::7ec2:55ff:fe14:f11c/64
[nic] [3] eno2 :: 
[nic] [4] enp195s0f0 :: 
[nic] [5] enp195s0f1 :: 
[nic] [6] bond0 :: 10.254.130.115/24, fe80::ac0:ebff:fe3c:1948/64
[cuda-env]: CUDA_PATH=/data/apps/cuda/11.1
[cuda-env]: CUDA_HOME=/data/apps/cuda/11.1
[cuda-env]: CUDA_VISIBLE_DEVICES=0,1
[cuda-env]: CUDA_ROOT=/data/apps/cuda/11.1
[I] will parallel run 2 instances of /HOME/scz1075/.conda/envs/kungfu/bin/python3 with ["tf_cnn_benchmarks.py" "--batch_size=64" "--data_format=NCHW" "--model=resnet50" "--optimizer=momentum" "--variable_update=kungfu" "--kungfu_option=sync_sgd" "--nodistortions" "--num_gpus=1" "--num_epochs=90" "--weight_decay=1e-4" "--data_dir=/HOME/scz1075/run/data/cifar-10-batches-py"]
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:58:55.082500: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:58:55.082502: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
[[1;32m127.0.0.1.10000[m::stdout] KungFu does not require to call init()
[[1;34m127.0.0.1.10001[m::stdout] KungFu does not require to call init()
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:39.525150: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:39.525150: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:39.637989: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcuda.so.1
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:39.637989: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcuda.so.1
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:39.758274: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1733] Found device 0 with properties: 
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] pciBusID: 0000:c1:00.0 name: NVIDIA GeForce RTX 3090 computeCapability: 8.6
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] coreClock: 1.695GHz coreCount: 82 deviceMemorySize: 23.69GiB deviceMemoryBandwidth: 871.81GiB/s
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:39.758267: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1733] Found device 0 with properties: 
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] pciBusID: 0000:e1:00.0 name: NVIDIA GeForce RTX 3090 computeCapability: 8.6
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] coreClock: 1.695GHz coreCount: 82 deviceMemorySize: 23.69GiB deviceMemoryBandwidth: 871.81GiB/s
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:39.777854: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:39.777852: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:40.414639: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcublas.so.11
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:40.414767: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcublasLt.so.11
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:40.414631: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcublas.so.11
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:40.414774: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcublasLt.so.11
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:40.593301: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcufft.so.10
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:40.593301: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcufft.so.10
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:40.807196: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcurand.so.10
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:40.807195: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcurand.so.10
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:41.081362: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcusolver.so.11
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:41.081443: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcusolver.so.11
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:41.148816: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcusparse.so.11
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:41.148816: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcusparse.so.11
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:41.234880: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudnn.so.8
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:41.234859: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudnn.so.8
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:41.251941: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1871] Adding visible gpu devices: 0
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:41.251940: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1871] Adding visible gpu devices: 0
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:41.251989: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:41.252010: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:44.117507: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1258] Device interconnect StreamExecutor with strength 1 edge matrix:
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:44.117513: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1258] Device interconnect StreamExecutor with strength 1 edge matrix:
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:44.117562: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1264]      0 
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:44.117561: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1264]      0 
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:44.117570: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1277] 0:   N 
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:44.117572: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1277] 0:   N 
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] 2024-10-14 18:59:44.122346: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1418] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 22299 MB memory) -> physical GPU (device: 0, name: NVIDIA GeForce RTX 3090, pci bus id: 0000:e1:00.0, compute capability: 8.6)
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] 2024-10-14 18:59:44.122391: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1418] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 22300 MB memory) -> physical GPU (device: 0, name: NVIDIA GeForce RTX 3090, pci bus id: 0000:c1:00.0, compute capability: 8.6)
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] Traceback (most recent call last):
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py", line 64, in <module>
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] Traceback (most recent call last):
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py", line 64, in <module>
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]     app.run(main)  # Raises error on invalid flags, unlike tf.app.run()
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]   File "/HOME/scz1075/.conda/envs/kungfu/lib/python3.9/site-packages/absl/app.py", line 312, in run
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]     app.run(main)  # Raises error on invalid flags, unlike tf.app.run()
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]   File "/HOME/scz1075/.conda/envs/kungfu/lib/python3.9/site-packages/absl/app.py", line 312, in run
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]     _run_main(main, args)
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]   File "/HOME/scz1075/.conda/envs/kungfu/lib/python3.9/site-packages/absl/app.py", line 258, in _run_main
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]     _run_main(main, args)
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]   File "/HOME/scz1075/.conda/envs/kungfu/lib/python3.9/site-packages/absl/app.py", line 258, in _run_main
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]     sys.exit(main(argv))
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py", line 50, in main
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]     sys.exit(main(argv))
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py", line 50, in main
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]     bench = benchmark_cnn.BenchmarkCNN(params)
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py", line 1250, in __init__
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]     bench = benchmark_cnn.BenchmarkCNN(params)
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/benchmark_cnn.py", line 1250, in __init__
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]     self.dataset = dataset or datasets.create_dataset(self.params.data_dir, self.params.data_name)
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/datasets.py", line 245, in create_dataset
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]     self.dataset = dataset or datasets.create_dataset(self.params.data_dir, self.params.data_name)
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]   File "/data/run01/scz1075/kf-benchmarks/scripts/tf_cnn_benchmarks/datasets.py", line 245, in create_dataset
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m]     raise ValueError('Could not identify name of dataset. '
[[1;32m127.0.0.1.10000[m::[1;35mstderr[m] ValueError: Could not identify name of dataset. Please specify with --data_name option.
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m]     raise ValueError('Could not identify name of dataset. '
[[1;34m127.0.0.1.10001[m::[1;35mstderr[m] ValueError: Could not identify name of dataset. Please specify with --data_name option.
[1;35m[E][m #<127.0.0.1.10001> exited with error: exit status 1
[1;35m[E][m #<127.0.0.1.10000> exited with error: exit status 1
[I] all 2/2 local peers finished, took 1m15.342040408s
exit on error: 2 tasks failed at 47373690513931:/data/run01/scz1075/KungFu/srcs/go/kungfu/runner/simple.go:19
